package.xml

```diff
@@ -35,6 +35,7 @@
   <build_depend>tf2_ros</build_depend>
   <build_depend>tf2_eigen</build_depend>
   <build_depend>tf2_geometry_msgs</build_depend>
+  <build_depend>control_msgs</build_depend>
```

Comment:
Don't we need `moveit_servo` as a build/run depend here too?

Reply:
Yep! Fixed